backend/src/app/modules/course/course.service.ts

```
@@ -133,10 +135,22 @@ const getAllCourses = async (filters: any) => {
         }
     }
-    const result = await prisma.course.findMany({
-        include: includeTerms
-
+    const whereTerms = {
+        instructorId: payload.filters?.instructorId,
+        title: {
+            contains: payload.filters?.title,
+        }
+    }
+    let sortBy = [] as any
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'sortBy' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)